pkg/trace/config/apply.go

```diff
@@ -391,6 +416,24 @@ func readTraceWriterConfigYaml() writerconfig.TraceWriterConfig {
 	return c
 }
 
+func readFeaturesConfigYaml() featuresconfig.FeaturesConfig {
+	w := features{}
+	c := featuresconfig.DefaultFeaturesConfig()
+
+	if err := config.Datadog.UnmarshalKey("apm_config.features", &w); err == nil {
```

Comment:
what is this config used for?
I can't seem to find any change to the agent config in this pr

Reply:
Same reading optional config object thing, which is why it's missing from the agent config. 

This just serves as an override to one of these: 
```
// HTTPRequestTimeoutDuration is the HTTP timeout for POST requests to the StackState backend
	HTTPRequestTimeoutDuration   time.Duration
	// FeatureRequestTickerDuration is the duration in between each of these requests
	FeatureRequestTickerDuration time.Duration
	// MaxRetries is the maximum number of times we'll try to fetch features from StackState
	MaxRetries                   int
```

Reply:
ok